science/SoapySDRPlay/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.0
+
+github.setup        pothosware SoapySDRPlay 0.2.0 soapy-sdrplay-
+
+name                SoapySDRPlay
+platforms           darwin macosx
+categories          science
+license             MIT
+maintainers         {@ra1nb0w irh.it:rainbow} {michaelld @michaelld} openmaintainer
+
+description         Soapy SDR module for SDRPlay
+long_description    ${description}
+
+checksums           rmd160  d1ffd21319d7ffe5604ab6e4ff433da31e88eb3d \
+                    sha256  58ba22649564c403215ef568e106b5f2ac5a690845718ddd029960bbbb95a88c \
+                    size    13566
+
+homepage            https://github.com/pothosware/SoapySDRPlay/wiki
```

Comment:
Normally we just use the default homepage set by the github portgroup.

Reply:
Good to know and in this case could be a good thing. Done